build/root/usr/local/bin/ci_entrypoint_gpu-operator.sh

```diff
@@ -81,6 +90,44 @@ test_commit() {
                                              --tag_uid="${CI_IMAGE_GPU_COMMIT_CI_IMAGE_UID}"
 
     ./run_toolbox.py gpu_operator deploy_from_bundle "--bundle=${GPU_OPERATOR_QUAY_BUNDLE_IMAGE_NAME}:operator_bundle_gpu-operator-ci-image"
+}
+
+test_alerts() {
+    gpu_operator_git_repo="${1:-}"
+    gpu_operator_git_ref="${2:-}"
+
+    oc label ns/openshift-operators openshift.io/cluster-monitoring=true --overwrite
+
+    deploy_commit $gpu_operator_git_repo $gpu_operator_git_ref
+
+    ./run_toolbox.py gpu-operator prepare_test_alerts \
+                     --alert_delay=1 \
+                     --alert_prefix=CI
+
+    # wait for NFD alert
+    ./run_toolbox.py cluster wait_for_alert \
+                     CIGPUOperatorReconciliationFailedNfdLabelsMissing \
+                     --alert-active=true
+
+    ./run_toolbox.py nfd_operator deploy_from_operatorhub
+    ./run_toolbox.py cluster set_scale g4dn.xlarge 1
+
+    # wait for NFD alert to go off
```

Comment:
An "alert going off" means the exact opposite of what you are waiting for here :)

Reply:
oh you mean in English? shoot, so "go on" and "go off" means the same 🙃

Reply:
I'll rewrite it then